mimic/rest/maas_api.py

```diff
@@ -354,6 +359,31 @@ def parse_and_flatten_qs(url):
     return flat_qs
 
 
+def _random_hipsum():
+    """
+    Generates a random sentence using Hipsum ( http://hipsum.co/ ).
```

Comment:
This should document its return value and type: is this bytes or text? If it's text the values below should be `unicode` objects; otherwise they should have a `b` prefix on them.